portal/templates/profile_macros.html

```diff
@@ -1687,7 +1688,21 @@ <h4 id="communicationsLoc" class="profile-item-title index-item">{{ _("Communica
                 </div>
             </div>
         </div>
-    {%- endif %}
+    {%- elif person and person.has_role(ROLE.STAFF) and current_user and current_user.has_role(ROLE.STAFF_ADMIN)-%}
```

Comment:
Ah, Ivan, I see this is confusing because the person here refers to the person whose profile is being viewed while the current_user is the person viewing the profile.  So what I want to accomplish here is to allow the staff_admin to view the communications section of the person that has the role of staff.